easybuild/easyconfigs/m/MitoFinder/MitoFinder-1.4.1-foss-2022b-Python-2.7.18.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'Tarball'
+
+name = 'MitoFinder'
+version = '1.4.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/RemiAllio/MitoFinder'
+description = """MitoFinder is a pipeline to assemble mitochondrial genomes and annotate mitochondrial genes from trimmed
+read sequencing data"""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+source_urls = ['https://github.com/RemiAllio/MitoFinder/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['c80a0108c64813dce8c3f653958ed5d6bddc28a0c801fe4d477a173c3a58fe45']
+
+dependencies = [
+    ('Java', '11', '', SYSTEM),
+    ('Python', '2.7.18', '-bare'),
+]
+
+postinstallcmds = ['./install.sh']
```

Comment:
install script does a nasty `chmod` to start with, and then also installs a couple of dependencies from source, which we should try to do with EasyBuild instead?